src/test/features/claimant-response/helpers/settlementHelper.ts

```diff
@@ -49,7 +49,7 @@ describe('settlementHelper', () => {
       expect(partyStatement.madeBy).to.be.eql('DEFENDANT')
       expect(partyStatement.type).to.be.eql('OFFER')
       expect(partyStatement.offer).is.not.undefined
-      expect(partyStatement.offer.content).to.be.eql('John Smith will pay the full amount, no later than 31 December 2050')
+      expect(partyStatement.offer.content).to.be.eql('John Smith will pay £200, no later than 31 December 2050')
```

Comment:
hardcoded ? take from test data?

Reply:
Can this review comment be addressed so that I can approve PR?

Reply:
@pragnesh-patel the value is derived and it cant be retrieved from the test data. So only we can check this using hard coded value.